tests/layer_tests/pytorch_tests/test_einsum.py

```diff
@@ -51,8 +51,8 @@ def forward(self, x):
     @pytest.mark.nightly
     @pytest.mark.precommit
     @pytest.mark.xfail(reason='OpenVINO CPU plugin does not support einsum diagonal')
-    def test_einsum_batch_diagonal(self, ie_device, precision, ir_version):
-        self._test(*self.create_model(), ie_device, precision, ir_version, dynamic_shapes=False)
+    def test_einsum_batch_diagonal(self, ie_device, precision, ir_version, use_ts_backend):
+        self._test(*self.create_model(), ie_device, precision, ir_version, dynamic_shapes=False, use_ts_backend=use_ts_backend)
```

Comment:
@mvafin, can you recommend a better way to extend signature to avoid such a massive change and to make it more future proofed solution, like kwargs or something like that. With the currently extended approach we will add another parameter, like `use_fx_decoder` in the future and it will be again a bulky change in all the tests.

It is also a bit surprising to see such parameters like `ir_version` and others. Can we avoid explicitly trampolining data that are not required for a test?

Reply:
Agree, we need to process such parameters as device, precision, version, etc as kwargs

Reply:
@suryasidd, you mentioned that it would be fixed. If you have issues with that, please share your experience.

Reply:
Added env variable for USE_TS_BACKEND